programs/ssl/ssl_server2.c

```diff
@@ -329,6 +330,11 @@ int main( void )
     "    server_addr=%%s      default: (all interfaces)\n"  \
     "    server_port=%%d      default: 4433\n"              \
     "    debug_level=%%d      default: 0 (disabled)\n"      \
+	"    buffer_size=%%d      default: 200 \n" \
+    "                         (minimum: 1, max: 16385)\n" \
+    "    response_size=%%d    default: about 152 (basic response)\n" \
```

Comment:
I think that a default value should be something specific. If the "default" `response_size`'s known at the compile time I'd put it here. Otherwise I would not use the "default" word.

Reply:
It's just a "Hello World" type of a response that our server returns, indicating a succesful connection and listing which ciphersuite has been used. Even changing the ciphersuite will change the size of the response, since the printed string will be different (so it's not a compile-time-constant). 
The default size means that if we run the client and server with default parameters (or rather - without any) and config, the size will be just like mentioned. Would you like to remove the "default" instead?